be/src/exprs/bitmap_function.cpp

```diff
@@ -117,11 +337,181 @@ StringVal BitmapFunctions::bitmap_serialize(FunctionContext* ctx, const StringVa
     return result;
 }
 
+template<typename T, typename ValType>
+void BitmapFunctions::bitmap_intersect_init(FunctionContext* ctx, StringVal* dst) {
+    dst->is_null = false;
+    dst->len = sizeof(BitmapIntersect<T>);
+    auto intersect = new BitmapIntersect<T>();
+
+    // constant args start from index 2
+    for (int i = 2; i < ctx->get_num_constant_args(); ++i) {
+        DCHECK(ctx->is_arg_constant(i));
```

Comment:
Is it checked that arguments should be constant in Frontend? If it isn't checked, something bad will happen if user give a illegal query.

Reply:
done